x-pack/plugins/security_solution/common/detection_engine/types.ts

```diff
@@ -17,3 +17,5 @@ export const RuleTypeSchema = t.keyof({
   machine_learning: null,
 });
 export type RuleType = t.TypeOf<typeof RuleTypeSchema>;
+
+export type AlertStateStatus = 'open' | 'closed' | 'in-progress';
```

Comment:
You don't want to duplicate the types from `scheams.ts` but rather you can use the type from above of `Status`.

That's the pro of using io-ts and types like this is that there should only be one spot we update with a new enumeration type.